drivers/misc/ft8xx/ft8xx_copro.c

```diff
@@ -73,6 +78,183 @@ void ft8xx_copro_cmd_swap(void)
 	ft8xx_copro_cmd(CMD_SWAP);
 }
 
+void ft8xx_copro_cmd_fgcolor(uint32_t c)
```

Comment:
is c for `color` ? would use the full term :)

Reply:
I think so. It's how the programming guide defines it:
![image](https://github.com/user-attachments/assets/30661975-179d-44b2-b548-ed7b0743ad01)

I would prefer to keep functions prototypes like in the programming guide.